hibernate-reactive-core/src/main/java/org/hibernate/reactive/pool/impl/SqlClientConnection.java

```diff
@@ -199,45 +204,87 @@ private void feedback(String sql) {
 	}
 
 	private SqlClient client() {
-		return transaction != null ? transaction : connection;
+		return connection;
```

Comment:
Confused: how can this change be right?

Reply:
Gosh.. you are right.
Not seeing any failures in the tests made me beleive that everything was working.
I will work on it

Reply:
Interesting... we actually some failures but only with DB2